style.css

```diff
@@ -0,0 +1,100 @@
+@import 'https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap';
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  background: #f1f1f1;
+}
+
+.main-header {
+  height: 50px;
+}
```

Comment:
Every browser applies certain styles especially properties like `height` and `width` so each HTML element takes the size of its own content, this will help a lot with `responsiveness` which is a topic you will read about in the coming lessons.

- [ ] I highly recommend removing the `height` property and using padding or margins for this container instead to achieve the wished style.